modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -647,6 +707,7 @@ public void testGetIdPFederatedAuthenticators() throws Exception {
                 .assertThat()
                 .statusCode(HttpStatus.SC_OK)
                 .body(baseIdentifier + "name", equalTo("GoogleOIDCAuthenticator"))
```

Comment:
check authenticatorId as well.

Reply:
This is updated with 7831a169607232cefc26307b20c8ca3746542a52